docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,443 @@
+   ```
+   export FABRIC_CA_CLIENT_HOME=<RELATIVE-PATH-TO-org1-ca-FOLDER>/org1-ca/admin
+   ```
+   For example, if the `org1-ca` folder resides inside the Fabric CA Client directory it would simply be:
+   ```
+   export FABRIC_CA_CLIENT_HOME=org1-ca/admin
+   ```
+
+5. Now you can use the Fabric CA client to generate the CA admin certificate and private key. You need this certificate and private key to be able to issue identities using this CA. Run the command:
+   ```
+   fabric-ca-client enroll -d -u https://<ADMIN>:<ADMIN-PWD>@<CA-URL>:<PORT> --tls.certfiles <RELATIVE-PATH-TO-TLS-CERT> --csr.hosts '<CA_HOSTNAME>'
+   ```
+   Replace:
+   - `<ADMIN>` - with the organization CA admin specified on the `init` command.
+   - `<ADMIN-PWD>` - with the organization CA admin specified on the `init` command.
+   - `<CA-URL>` - with the hostname specified in the `csr` section of the organization CA configuration .yaml file.
+   - `<PORT>` - with the port that the organization CA is listening on.
+   - `<RELATIVE-PATH-TO-TLS-CERT>` - with the path to the ca-cert.pem file that you copied from your TLS CA. This is the path relative to `FABRIC_CA_CLIENT_HOME`.
+   - `<CA_HOSTNAME>` - with a comma-separated list of host names for which the certificate should be valid. If not specified, the default value from the `fabric-ca-client-config.yaml` is used. If a host name is dynamic you can specify a wildcard for the domain. For example, when you include the flag `--csr.hosts 'host1,*.example.com'` it means that the hostname `host1` is recognized as well as any host from the `example.com` domain.
+
+   In this case, the `-d` parameter runs the client in DEBUG mode which is useful for debugging command failures.
+
+   For example:
+   ```
+   fabric-ca-client enroll -d -u https://rcaadmin:rcaadminpw@my-machine.example.com:7055 --tls.certfiles ../crypto/ca-cert.pem --csr.hosts 'host1,*.example.com'
+   ```
+
+When this command completes, an `/msp` folder is generated under `/org1-ca/admin/msp` and contains the signing cert and private key for the org1 CA and looks similar to:
+
+```
+└── msp
+   ├── IssuerPublicKey
+   ├── IssuerRevocationPublicKey
+   ├── cacerts
+   │   └── my-machine-example-com-7055.pem
+   ├── keystore
+   │   └── 60b6a16b8b5ba3fc3113c522cce86a724d7eb92d6c3961cfd9afbd27bf11c37f_sk
+   ├── signcerts
+   │   └── cert.pem
+   └── user
+```
+
+Where:
+- `my-machine-example-com-7055.pem` is the **Organization CA root certificate**.
+- `60b6a16b8b5ba3fc3113c522cce86a724d7eb92d6c3961cfd9afbd27bf11c37f_sk` is the **private key** for the organization CA admin identity. This key needs to be protected and should not be shared with anyone. It is required to be able to register and enroll other identities with this CA.
+- `cert.pem` is the CA admin identity **signing certificate**.
```

Comment:
Let folks they need to leave the root running to use the intermediate.

Reply:
Done